src/components/UserLocationPin.js

```diff
@@ -0,0 +1,12 @@
+const UserLocationPin = () => (
```

Comment:
I’m not seeing the user’s location pin on Safari 🤔 

![Screen Cast 2021-08-03 at 9 19 58 PM](https://user-images.githubusercontent.com/4306/128121639-31a22bb8-8b0d-4769-9e92-7d6a617d4868.gif)

Reply:
Yeah, I don't see it on my mobile device (iOS, Safari browser) either.

Reply:
In fact, in Safari, I'm not finding it when I inspect, either.

Reply:
Missing in Firefox too.

Reply:
@laurenmbeatty, and @segdeha : Thanks to Lauren's review and article. I could find out the possible reason for this error. I'm so sorry. There is a browser support problem with the `navigator?.permissions`. I'm going to change to `navigator.geoLocation`, and think of a way to know the user start/stop sharing your location. Thanks for your reviews!